_data/civic_hackers.yml

```diff
@@ -26,6 +26,7 @@ Civic Hackers:
   - open-city
   - openaustralia
   - opendatakosovo
+  - girlscodingkosova
```

Comment:
@nschonni I've gone back and forth about the ordering of the list. I do think there is value in grouping like organizations/governments (I see this a lot in US state agencies). Almost makes me want to provide some additional ordering, even if it's not presented in the markup